scripts/restartdaq

```diff
@@ -64,6 +70,10 @@ do
 	esac
 done
 
+# Prevent ShellCheck SC2034 warning for unused vars passed to other scripts
+# shellcheck disable=SC2034
+: "$DAQMGR_CNF" "$VERBOSE"
```

Comment:
Can someone explain what this means to me?
To my eye, shellcheck's warning here is correct. This line is a no-op and these variables are both unused and not exported. So, how are they being passed to other scripts?
Is there a bash trick I don't know?